index.bs

```diff
@@ -181,6 +275,9 @@ shared among all [=environment settings objects=].
 
 User agents SHOULD place limits on the maximum [=list/size=] of the [=attribution caches=].
 
+User agents SHOULD periodically remove entries from the [=attribution source cache=] where the entry's
+[=attribution source/expiry=] is in the past.
```

Comment:
Good catch, but this makes me realize that some of this actually needs to be expressed in the algorithms that add entries to the attribution caches. So, below at the "Add |source| to cache." step, you should say something like "If the size of |cache| is less than an implementation-defined limit ..."

You could have it remove expired entries in that algorithm too, or this "periodically" is probably ok (with a different meaning).